.github/hooks/pre-commit

```diff
@@ -5,6 +5,12 @@
 # ln -s ../../.github/hooks/pre-commit .git/hooks/pre-commit
 #
 
+# Regenerate circle-ci config-yml
```

Comment:
:wrench: Couldn't we just do "npm run circleci:update-config", then we wouldn't have to duplicate the code?

Reply:
or better (imo :D ) just run `scripts/circleci-update-config.sh` to avoid spinning up npm which just runs `scripts/circleci-update-config.sh`.  Spinning up npm takes an additional 0.15s  🤡